apps/web/src/utils/date.ts

```diff
@@ -6,6 +6,14 @@ export function normalizeTimestamp(timestamp: number | Date | string) {
     : dayjs(timestamp);
 }
 
+export function getISODate(date: dayjs.Dayjs | Date | string) {
```

Comment:
should we add UTs here?

Reply:
Let's do component and e2e testing for the web app instead